mantidproject/mantid
# Add additional option to LoadPreNexusLive

Since there are multiple "live" files available, add an option to override which one is used (rather than just the most recent). The default behavior is still to load the most recent.

**To test:**

_Prerequisite:_ be at ORNL with the data mounts in place.
- Try loading live data using the defaults and get the latest run
- Try loading a specific live run number and see that you get that instead

_There is no associated issue_

_Does not need to be in the release notes_ because the algorithm is new to v3.8. The current release notes are still valid.

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
